pkg/proto/agentbaker/v1/config.proto

```diff
@@ -8,56 +8,58 @@ message Configuration {
   
   string cluster_certificate_authority = 1;
 
-  // only required until Secure TLS bootstrapping in place
-  // would use kubelet identity after that.
-  string tls_bootstrap_token = 2;
-
   // cluster/user config
-  string kubernetes_version = 3; // Q: can this be auto-detected? Or is this part of specifying the desired node version?
+  string kubernetes_version = 2; // Q: can this be auto-detected? Or is this part of specifying the desired node version?
```

Comment:
This is for node configuration at this point. Example: clean up of unused binaries, selection of corresponding addons based on k8s minor version